go.mod

```diff
@@ -1,6 +1,6 @@
 module go.infratographer.com/load-balancer-operator
 
-go 1.20
+go 1.21
```

Comment:
if you're going to bump this, you should hit the devcontainer Dockerfile and the project root Dockerfile